src/meta/configs.js

```diff
@@ -180,7 +180,7 @@ function updateConfig(config) {
 }
 
 function updateLocalConfig(config) {
-	Object.assign(Meta.config, config);
```

Comment:
Should probably just send the already-deserialized config between processes instead.

Reply:
I changed this line to `Object.assign(Meta.config, deserialize(config));` so when configs are saved in ACP they dont go back to strings.